scripts/review/diff.ts

```diff
@@ -28,10 +32,70 @@ function parseOptions(): Options {
   };
 }
 
+/**
+ *
+ * @param changes The changes in the diff patch.
+ * @param types The types of changes to check.
+ *
+ * This function checks if the changes in the diff patch are of the given types.
```

Comment:
```suggestion
 * Check if the changes in the diff patch are of the given types.
 *
 * @param changes The changes in the diff patch.
 * @param types The types of changes to check.
```

This is typically how JSdoc is formatted.

Reply:
Same for other JSDoc comments.

Reply:
Updated 👍🏾